### PR TITLE
Test/156/Implement karate tests to Update Order Status

### PR DESCRIPTION
**Pull request by ABbondar**
## Issues
#156  
## Summary of changes🔥
- Implemented Karate tests to "Update Order Status" feature.

### CODE REVIEW

**Review by DenysLiubchenko — CHANGES_REQUESTED** [review]
Look what I did in getOrdersHistoryByUserId.feature and rewrite in the same way. Therefore we would have no need to change our secrets and there will be much less code.

File: e2e/karate/src/test/resources/apis/orders/features/orders/testUpdateOrderStatus.feature [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      | status |  role     |
+      | 200    | 'MANAGER' |
+      | 403    | 'USER'    |
+
+
+  Scenario Outline: Update Order Status with Incorrect Status | BAD REQUEST
+    * if (<role> == 'MANAGER') {karate.set('credentials.username', 'manager@mail.com'); karate.set('credentials.password', 'Manager_1234');}
+    Given def authHeader = call read('classpath:karate-auth.js')
+    And path '/v1/orders', orderId, 'status'
+    And param orderStatus = 'COMPLETEDDDD'
```
Comment (DenysLiubchenko) [inline e2e/karate/src/test/resources/apis/orders/features/orders/testUpdateOrderStatus.feature]: You can add orderStatus to Examples table of previous scenario and reuse first scenario

File: e2e/karate/src/test/resources/apis/orders/features/orders/testUpdateOrderStatus.feature [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    Then match responseStatus == <status>
+
+    Examples:
+      | status |  role     |
+      | 400    | 'MANAGER' |
+
+  Scenario Outline: Update Order Status with Non-existing Order ID | NOT FOUND
+    * if (<role> == 'MANAGER') {karate.set('credentials.username', 'manager@mail.com'); karate.set('credentials.password', 'Manager_1234');}
+    Given def authHeader = call read('classpath:karate-auth.js')
+    And path '/v1/orders', nonExistingOrderId, 'status'
```
Comment (DenysLiubchenko) [inline e2e/karate/src/test/resources/apis/orders/features/orders/testUpdateOrderStatus.feature]: You can add order id to Examples table of first scenario and reuse first scenario

**Review by DenysLiubchenko — CHANGES_REQUESTED** [review]

File: e2e/karate/src/test/resources/apis/orders/features/orders/testUpdateOrderStatus.feature (hunk 1 above)
Comment (ABbondar) [reply e2e/karate/src/test/resources/apis/orders/features/orders/testUpdateOrderStatus.feature]: good idea

**Review by DenysLiubchenko — APPROVED** [review]

**Pull request merged**